client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -60,7 +60,7 @@ const displaySiteUrl = ( siteUrl: string ) => {
 	return siteUrl.replace( 'https://', '' ).replace( 'http://', '' );
 };
 
-const VisitDashboardItem = ( { site }: { site: SiteData } ) => {
+const VisitDashboardItem = ( { site }: { site: SiteExcerptData } ) => {
```

Comment:
The dashboard item links are broken right now because `slug` isn't included in the API response even though it's requested.